tests/platform/cloud/DisruptionSequenceGradeConfig.js

```diff
@@ -0,0 +1,28 @@
+/*!
+Copyright 2019 OCAD university
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+
+*/
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii");
+
+fluid.require("%gpii-universal");
+fluid.registerNamespace("gpii.tests.cloud.oauth2.settings");
+
+gpii.tests.cloud.oauth2.settings.config = {
+    configName: "gpii.config.cloudBased.development",
+    configPath: "%gpii-universal/gpii/configs/shared"
+};
+
+// Grade for "disruptions" that are also proper sequence grades.  Use the grade
+// for couch test harness for development config
+fluid.defaults("gpii.test.disruption.sequenceGrade", {
```

Comment:
This grade has already been defined in [gpii/node_modules/testing/src/CloudBasedOAuth2TestsUtils.js](https://github.com/klown/universal/blob/GPII-3333/gpii/node_modules/testing/src/CloudBasedOAuth2TestsUtils.js#L143-L146).

Reply:
Actually, I removed the definition from `CloudBasedOath2TestUtils.js` in an earlier revision.  It snuck back in due to a subsequent merge with master.  It should not be defined here; more on this below.

Reply:
Can we rename the file name (tests/platform/cloud/DisruptionSequenceGradeConfig.js) and the grade name `gpii.test.disruption.sequenceGrade` to include "settings" to indicate they are for testing /settings endpoints only.